plugin/src/Reconciler/reify.lua

```diff
@@ -40,6 +42,20 @@ local function addAllToPatch(patchSet, virtualInstances, id)
 	end
 end
 
+local function createInstance(instanceMap, className, properties)
+	-- Custom logic for MeshParts
+	-- see: https://github.com/rojo-rbx/rojo/issues/402
+	if className == "MeshPart" then
+		local okMeshId, meshId = decodeValue(properties.MeshId, instanceMap)
```

Comment:
We need to handle `okMeshId`, because the second return value will not be a mesh ID if this failed to decode.